### ISSUE TITLE
httpGet probes with a path without leading / not showing correctly

### ISSUE THREAD

**Issue by pallxk** [aptakube/aptakube#192]
### Current and expected behavior

Current behavior:

```
GET http://192.168.194.117:httpzxcv
```

<img width="399" alt="image" src="https://github.com/aptakube/aptakube/assets/2920525/bd78dca1-09b1-4e2a-a7a5-eb85556b4ec6">

 Expected behavior:

```
GET http://192.168.194.117:http/zxcv
```

### Environment & Operating System

Aptakube/1.6.0

**Comment by goenning** [comment]
hey @pallxk can you confirm what's in your spec?
Is it like this?

```yaml
      livenessProbe:
        httpGet:
          path: zxcv
          port: http
```

**Comment by pallxk** [comment]
Yes, exactly. The `httpGet.path` is specified without a leading `/`.

**Comment by goenning** [comment]
@joseluisq this should be fixed now on 1.6.1, but please reopen if it's not. Thanks for reporting this bug!
